src/themes/brutalist.css

```
@@ -0,0 +1,832 @@
+  --wa-font-family-longform: 'Lora', serif;
+
+  --wa-font-weight-light: 300;
+  --wa-font-weight-normal: 400;
+  --wa-font-weight-medium: 500;
+  --wa-font-weight-heavy: 700;
+
+  --wa-font-weight-heading: var(--wa-font-weight-medium);
+  --wa-font-weight-body: var(--wa-font-weight-normal);
+  --wa-font-weight-action: var(--wa-font-weight-medium);
+
+  /* The default typescale is based on the Major Second scale (x1.125).
+   * Every other step on the scale is skipped for larger sizes in order to maximize variation. */
+  --wa-font-size-root: 16px;
+  --wa-font-size-2xs: 0.6875rem; /* 11px */
+  --wa-font-size-xs: 0.75rem; /* 12px */
+  --wa-font-size-s: 0.875rem; /* 14px */
+  --wa-font-size-m: 1rem; /* 16px */
+  --wa-font-size-l: 1.25rem; /* 20px */
+  --wa-font-size-xl: 1.625rem; /* 26px */
+  --wa-font-size-2xl: 2rem; /* 32px */
+  --wa-font-size-3xl: 2.5625rem; /* 41px */
+  --wa-font-size-4xl: 3.25rem; /* 52px */
+
+  --wa-font-line-height-compact: 1.25;
+  --wa-font-line-height-regular: 1.6;
+  --wa-font-line-height-comfortable: 2;
+
+  /**
+   * Spacing
+   * Used intentionally, space properties yield a predictable rhythm and support effective implementation of the proximity principle.
+   * Space can be organized into three groups with distinct usage -
+   *  *  Small-scale space (3xs, 2xs, and xs) is used for gaps between closely related elements, such as a dropdown button and its menu,
+   *     and padding within small components, such as badges and tooltips
+   *  *  Normal space (s, m, and l) is used for gaps between related elements with distinct purposes or touch targets and padding within
+   *     typical interface elements, such as buttons and inputs
+   *  *  Large-scale space (xl, 2xl, and 3xl) is used for gaps between unrelated elements and padding within larger components,
+   *     such as cards and dialogs
+   */
+  /* Space is designed to scale according to a single base value.
+   * The base value is intended for calculations and is not used by components directly. */
+  --wa-space-base: 1;
+  --wa-space-3xs: calc(var(--wa-space-base) * 0.125rem); /* 2px */
+  --wa-space-2xs: calc(var(--wa-space-base) * 0.25rem); /* 4px */
+  --wa-space-xs: calc(var(--wa-space-base) * 0.5rem); /* 8px */
+  --wa-space-s: calc(var(--wa-space-base) * 0.75rem); /* 12px */
+  --wa-space-m: calc(var(--wa-space-base) * 1rem); /* 16px */
+  --wa-space-l: calc(var(--wa-space-base) * 1.25rem); /* 20px */
+  --wa-space-xl: calc(var(--wa-space-base) * 1.5rem); /* 24px */
+  --wa-space-2xl: calc(var(--wa-space-base) * 2rem); /* 32px */
+  --wa-space-3xl: calc(var(--wa-space-base) * 3rem); /* 48px */
+
+  /**
+   * Borders
+   */
+  --wa-border-style: solid;
+
+  /* Border widths are designed to scale according to a single base value.
+     * The base value is intended for calculations and is not used by components directly. */
+  --wa-border-width-base: 0.0625;
```

```suggestion
  --wa-border-width-base: 0.125;
```

Since we have heavier borders throughout this theme, we can change this property to make sure that we thicken up everything across the board.